app/helpers/peer_review_helper.rb

```diff
@@ -8,8 +8,7 @@ def create_map_reviewee_to_reviewers(reviewer_groups, reviewee_groups)
     peer_review_map = Hash.new { |hash, key| hash[key] = [] }
     reviewee_groups.each { |reviewee| peer_review_map[reviewee['id']] }
 
-    peer_reviews = PeerReview.where(reviewer_id: reviewer_ids)
-    peer_reviews.each do |peer_review|
+    PeerReview.includes(:result).where(reviewer_id: reviewer_ids).each do |peer_review|
```

Comment:
Added the required includes, and _wow_ this is insanely fast (probably as expected though).